tiatoolbox/utils/transforms.py

```diff
@@ -94,7 +94,7 @@ def imresize(img, scale_factor=None, output_size=None, interpolation="optimise")
         height = int(img.shape[0] * scale_factor[1])
         output_size = (width, height)
 
-    if scale_factor is None:
+    elif scale_factor is None:
```

Comment:
This could be just else

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/369/commits/aec15a336f84a63afce520b51b2faa1f43b00853